pkg/state/create.go

```diff
@@ -173,13 +173,13 @@ func (c *StateCreator) ParseAndLoad(content []byte, baseDir, file string, envNam
 			return nil, errors.New("nested `base` helmfile is unsupported. please submit a feature request if you need this!")
 		}
 	} else {
-		state, err = c.loadBases(envValues, state, baseDir)
+		state, err = c.loadBases(envValues, overrode, state, baseDir)
 		if err != nil {
 			return nil, err
 		}
 	}
 
-	state, err = c.LoadEnvValues(state, envName, envValues, evaluateBases)
+	state, err = c.LoadEnvValues(state, envName, envValues, overrode, evaluateBases)
```

Comment:
Note: `envValues` had been already incorporated overrides before this change. Now, we incorporate overrides within LoadEnvValues. Gotcha.

Reply:
However, the result of LoadEnvValues is now different- It applies overrides on top of the resulting Env.